### DISCUSSION TITLE
Add alarm real-time configuration display

### DISCUSSION THREAD

**Discussion by marksugar** [apache/skywalking#9097] [Ideas]
I using skywalking from 9.0
In kubernetes, when modify the configmap of alarm-settings.yml, I don't know when the alarm is applied, usually it takes time

have an idea, is it possible to add a button to the Alarm page to show  the configuration status of alarm-settings.yml

Of course, the configuration shown is the one currently in use

what do you think

thanks

**Comment by wu-sheng** [comment]
It wouldnot work, OAP is working in cluster, and UI would call one OAP pod randomly.

**Reply by marksugar** [reply]
I did not find the parameters and configuration of hot update
That means, updating the alarm-settings.yml configuration file can only take effect immediately after restarting？

**Reply by wu-sheng** [reply]
Dynamic configuration is there for alarm rules and other configurations update, it is not updating a YAML file. This is a wrong practice in a distributed system.

**Reply by marksugar** [reply]
i was delayed some time
---------
Need a dynamic way to debug thresholds
I now restart every time it takes effect
it's not friendly
is there a better way @wu
Grateful

**Reply by wu-sheng** [reply]
It seems you still didn't read the docs about **Dynamic configuration**. I don't know why you are willing to wait 5 days but not put the keyword on the website to search in 1 second.

This is the last time I replied to this thread, as you didn't follow my reply.

https://skywalking.apache.org/docs/main/latest/en/setup/backend/dynamic-config/#dynamic-configuration
